README.md

```diff
@@ -37,50 +37,7 @@ Build tasks are provided as [Taskfiles](https://taskfile.dev/#/). A full list of
 available tasks can be shown by running `./vendor/bin/task --list` (or
 `ddev task --list` if you're using DDEV).
 
-### Drupal
```

Comment:
Why are we removing these docs?

Reply:
they're not covering everything and get out of sync quickly, so better to just tell the user to run `--list` for now imo